pkg/archive/archive.go

```diff
@@ -103,6 +115,7 @@ func xzDecompress(archive io.Reader) (io.ReadCloser, error) {
 	return CmdStream(exec.Command(args[0], args[1:]...), archive)
 }
 
+// DecompressStream decompress the archive.
```

Comment:
Maybe you could add that it returns a ``Reader`` (or a ``ReadCloser``) with the decompressed archive.